tests/test_override_system_js_lib_symbol.js

```diff
@@ -7,7 +7,7 @@ mergeInto(LibraryManager.library, {
 	glTexImage2D__deps: ['orig_glTexImage2D'],
 	glTexImage2D: function(target, level, internalFormat, width, height, border, format, type, pixels) {
 		_glTexImage2D.createdType = type;
-		_orig_glTexImage2D(target, level, internalFormat, width, height, border, format, type, pixels);
+		assert(_orig_glTexImage2D);
```

Comment:
This does reduce the test's power from running the orig method and seeing that it operates correctly, to just seeing that it is truthy, IIUC?

I do agree moving this out of the browser suite is a reasonable idea. Perhaps the test can use something other than WebGL then, that is runnable in the shell? Or, maybe there is a way to do more than assert on the truthiness of the orig method, like check it's contents or run it and get an appropriate error on not having GL.

Reply:
I think its fine just to test for existence of the method we are overriding..  I can't see any need to actually run it.

Reply:
I did think about doing this with a different library, but it would have to be one of the ones that we don't include when `AUTO_JS_LIBRARIES=0`: 

https://github.com/emscripten-core/emscripten/blob/94b70c51b9dcbf7b4868b5ef82dc2b201c576b75/src/modules.js#L119-L130

Most of those are also kind of graphics based so hard to actually invoke.  I guess I could use `library_uuid` or `library_async`.. but I', not sure its worth it.

Reply:
Why is running it not useful? That verifies the entire functionality end to end. I can imagine that saving us from bugs some day. For example, maybe we'll do a rewrite and mess this up and the orig method will end up identical to the override, leading to an infinite recursion if we run it, but the new test would mistakenly pass?

Reply:
I guess its just hard to imagine that `LibraryManager.library.glTexImage2D` could pre-exist.. but not be coming from the `library_webgl`... how else would that method pre-exist before this file is loaded?     That seems like a hard bug to imagine.

Reply:
(I guess maybe it could happen if this user library was processed twice...?)

Reply:
Yes, something like that maybe. We have two methods with the same name, one overriding the other, so there is a scary risk of them being identical if one overwrites the other. Or if we mess up the logic for creating `orig_FOO` and write the override there.

Personally I think it pays to be paranoid with this stuff...

Reply:
Nice test!